src/components/input-field/input-field.tsx

```diff
@@ -260,6 +260,11 @@ export class InputField {
             'mdc-text-field--required': this.required,
         };
 
+        const labelClassList = {
+            'mdc-floating-label textarea-label': true,
```

Comment:
I would like this tiny change, just to make it a little easier to read:

```suggestion
            'mdc-floating-label': true,
            'textarea-label': true,
```